cudaungappedextender/include/claraparabricks/genomeworks/cudaungappedextender/cudaungappedextender.hpp

```diff
@@ -0,0 +1,92 @@
+/*
+* Copyright 2020 NVIDIA CORPORATION.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#pragma once
+
+namespace claraparabricks
+{
+namespace genomeworks
+{
+namespace cudaungappedextender
+{
+
+struct Anchor
```

Comment:
I would name this `Seed` instead of `Anchor` to avoid confusion with the `Anchor` in cudamapper. It is almost the same thing, but in the end cudamapper's Anchor is different, because it also contains the read_id information. `Seed` also seems to be the vocabulary used by Sneha and Yatish.

Reply:
Can we have this as one common data structure re-used by both these elements? Since cudamapper also needs it we can make this a member of the Anchor struct. And yes, I just copied cudamapper's Anchor here for illustration.

Reply:
Since they are so tiny, I would just leave them separate for now. If we happen to see any benefits later on, we may still combine them.